scss/civicrm/common/_modals.scss

```diff
@@ -48,18 +47,16 @@
       background: none;
       font-size: $font-size-h2;
       font-weight: $crm-font-weight-h2;
+      line-height: $font-size-h1;
```

Comment:
I think using this variable might produce difficulties in future. Lets imagine we change the font-size for h1, that should not affect the line-height of the h2 in modal. I suggest to have a separate variable something like `$modal-header-height` or something like that whichever is consistent.